timesketch/lib/analyzers/sessionizer.py

```diff
@@ -0,0 +1,69 @@
+"""Sessionizing sketch analyzer plugin."""
+
+from __future__ import unicode_literals
+
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+
+
+class SessionizerSketchPlugin(interface.BaseSketchAnalyzer):
+    """Sessionizing sketch analyser. All events in sketch with id sketch_id
+    are grouped in sessions based on the time difference between them. Two
+    consecutive events are in the same session if the time difference between
+    them is less or equal then max_time_diff_micros"""
+
+    NAME = 'sessionizer'
+    max_time_diff_micros = 300000000
```

Comment:
I opt for making this configurable. This can wait until the user input to analyzers are ready, but maybe add a TODO so we remember it?

Reply:
We are planning on changing this in the next pull request, however we will add a to-do.

Reply:
Great!